src/core/dbus.c

```diff
@@ -979,6 +989,19 @@ int bus_init_private(Manager *m) {
         return 0;
 }
 
+static int bus_track_stash(sd_bus_track *t, char ***l) {
+        for (const char *n = sd_bus_track_first(t); n; n = sd_bus_track_next(t)) {
+                int c;
+
+                c = sd_bus_track_count_name(t, n);
```

Comment:
hmm, so subscription is not recursive, we always pass "false" in the "recurse" parameter when allocating the bus track object.

hence sd_bus_track_count_name() should return 1 always here, hence you can smplify this...

Reply:
I'll do the generic bus_track_to_strv thing then I guess.